src/components/Menu.js

```diff
@@ -52,6 +64,26 @@ class Menu extends Component {
           text="React Puzzle Games - 15 Puzzle"
         />
         <ToolbarGroup>
+          {/*
+          <RaisedButton
+            className="menuButton"
+            label="New game"
+            onTouchTap={onResetClick}
+            icon={<New className="menuIcon" />}
+          />
+        */
+          }
+          <RaisedButton
+            className="menuButton"
+            label={gameState === GAME_PAUSED ? 'Continue' : 'Pause Game'}
```

Comment:
"Pause" is enough, I think, it should be obvious you're pausing the game, so it doesn't need to be in the label.